include/tvm/meta_schedule/search_strategy.h

```diff
@@ -88,14 +88,17 @@ class SearchStrategyNode : public runtime::Object {
 
   /*!
    * \brief Pre-tuning for the search strategy.
+   * \param max_trials The maximum number of trials.
+   * \param num_trials_per_iter The number of trials per iteration.
    * \param design_spaces The design spaces used during tuning process.
```

Comment:
I noticed that we use plural here for `design_spaces`, I kind of prefer to use singular term for consistency, but no strong opinion.

Reply:
in fact, it's pretty consistent in the codebase - when it's a list, we add an `s`